src/cg_snap.c

```diff
@@ -278,9 +278,7 @@ static void PmoveSingle(void)
     PM_AirMove();
   }
 
-  // TODO: calc yaw similar to cgaz
-  float const yaw = (s.pm_ps.viewangles[YAW] * 65536 / 360) +
-                    RAD2SHORT(atan2f((float)-s.pm.cmd.rightmove, (float)s.pm.cmd.forwardmove));
+  float const yaw = RAD2SHORT(atan2f(s.wishvel[1], s.wishvel[0]));
```

Comment:
Could you add a `TODO` that we should not go to short angles just yet and keep using floating point numbers until we start drawing. Otherwise the positions of the snapzone borders will be slightly off since we round/snap the yaw angle here.